src/lib/services/tx/index.ts

```diff
@@ -390,6 +391,17 @@ export const useTxsSequencer = (limit = 10) => {
   };
 };
 
+export const useTxsCountSequencer = () => {
+  const endpoint = useLcdEndpoint();
+  const { currentChainId } = useCelatoneApp();
+
+  return useQuery(
+    [CELATONE_QUERY_KEYS.TXS_COUNT_SEQUENCER, endpoint],
+    async () => getTxsCountSequencer(endpoint, currentChainId),
```

Comment:
I thought we are gonna use this function as a fallback for sequencer overview stats

Reply:
I used this approach because the fallback query hasn't some properties like `latest_block`.